PSI_Tracker.js

```diff
@@ -0,0 +1,644 @@
+              let array = [];
+              // Go through all the Dimensions listed in the header and build the array
+              for (let j = 0; j < results_header[0].length; j++){
+                let dimension = results_header[0][j];
+                if(dimension in output){
+                  array.push(output[dimension])
+                }
+                else{
+                  array.push("")
+                  debug += "💢 Error on " + dimension.toString() + " - No data" + " | " 
+                }
+              }
+              // Append the array at the end of "Results"
+              sheet_results.appendRow(array)
+            }
+            else if(api == "Accessibility"){
+              let sheet_results = ACCESSIBILITY_SHEET;
+              let results_header = ACCESSIBILITY_SHEET_HEADER;
+              // We translate the dic output to an array row based on indexes found in "Results"
+              let array = [];
+              // Go through all the Dimensions listed in the header and build the array
+              for (let j = 0; j < results_header[0].length; j++){
+                let dimension = results_header[0][j];
+                if(dimension in output){
+                  array.push(output[dimension])
+                }
+                else{
+                  // We ultimately decided to leave blank columns without data and leave the details in the debugging.
+                  // array.push("No data")
+                  array.push("")
+                  debug += "💢 Error on " + dimension.toString() + " - No data" + " | " 
+                }
+              }
+              // Append the array at the end of "Results"
+              sheet_results.appendRow(array)
+            }
+            else if(api == "Sustainability"){
+              let sheet_results = SUSTAINABILITY_SHEET;
+              let results_header = SUSTAINABILITY_SHEET_HEADER;
+              // We translate the dic output to an array row based on indexes found in "Results"
+              let array = [];
+              // Go through all the Dimensions listed in the header and build the array
+              for (let j = 0; j < results_header[0].length; j++){
+                let dimension = results_header[0][j];
+                if(dimension in output){
+                  array.push(output[dimension])
+                }
+                else{
+                  // We ultimately decided to leave blank columns without data and leave the details in the debugging.
+                  // array.push("No data")
+                  array.push("")
+                  debug += "💢 Error on " + dimension.toString() + " - No data" + " | " 
+                }
+              }
+              // Append the array at the end of "Results"
+              sheet_results.appendRow(array)
+            }
+
+
+            // STEP 4. SCREENSHOTS
+            if(save_screenshot){
+              let array_screenshot = [
```

Comment:
This can be const, since you use `push()` and `concat()` below, but never change the array being referenced.

Reply:
done!